web/static/js/gravatar.js

```diff
@@ -0,0 +1,22 @@
+const md5 = require('blueimp-md5');
+const inputSelector = '[data-javascript=gravatar-email]';
+const imageSelector = '[data-javascript=gravatar-display]';
+$(function() {
```

Comment:
Not nitpicky, I' not sure why the line breaks are crazy like that. Thanks for the catch!